ulogme_serve.py

```diff
@@ -67,8 +67,8 @@ def do_POST(self):
     self.end_headers()
     self.wfile.write(result)
 
-httpd = SocketServer.ThreadingTCPServer((IP, PORT), CustomHandler)
+httpd = socketserver.ThreadingTCPServer((IP, PORT), CustomHandler)
 
-print 'Serving ulogme, see it on http://localhost:' + `PORT`
+print('Serving ulogme, see it on http://localhost:',str(PORT))
```

Comment:
For it to be a link in the terminal you have to use `+` instead of `,`:
```python
print('Serving ulogme, see it on http://localhost:'+str(PORT))
```
or even better, with [F-Strings](https://www.python.org/dev/peps/pep-0498/) (but they depend on Python 3.6):

```python
print(f'Serving ulogme, see it on http://localhost:{PORT}'))
```